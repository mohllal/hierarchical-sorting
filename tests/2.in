property0|property1|property2|net_sales
bar|bro|jok|200
foo|sauce|scoue|300
foo|sauce|tyri|500
bar|sup|$total|-100
bar|sup|tre|-400
bar|bro|nbnu|400
$total|$total|$total|1600
bar|$total|$total|1100
foo|sauce|$total|800
foo|bacon|cre|100
bar|sup|vbhi|300
foo|bacon|$total|300
foo|bacon|hji|200
bar|$total|$total|500
bar|bro|$total|600