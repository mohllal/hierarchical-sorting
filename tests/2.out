property0|property1|property2|net_sales
$total|$total|$total|1600
foo|$total|$total|1100
foo|sauce|$total|800
foo|sauce|tyri|500
foo|sauce|scoue|300
foo|bacon|$total|300
foo|bacon|hji|200
foo|bacon|cre|100
bar|$total|$total|500
bar|bro|$total|600
bar|bro|nbnu|400
bar|bro|jok|200
bar|sup|$total|-100
bar|sup|vbhi|300
bar|sup|tre|-400